File: templates/app/skeleton/backend/gofiber/internal/handler/features.go
package handler

import (
	"github.com/gofiber/fiber/v2"
	"gorm.io/gorm"
)

// FeaturesResponse reports which optional features are enabled.
type FeaturesResponse struct {
	Database bool `json:"database"`
}

// FeaturesHandler provides the features endpoint.
type FeaturesHandler struct {
	DB *gorm.DB
}

// Features godoc
// @Summary     List enabled features
// @Tags        ops
// @Produce     json
// @Success     200 {object} FeaturesResponse
// @Router      /api/features [get]
func (h *FeaturesHandler) Features(c *fiber.Ctx) error {
	return c.JSON(FeaturesResponse{
		Database: h.DB != nil,
	})
}

File: templates/app/skeleton/backend/fastapi/app/handler/features.py
"""Features endpoint — reports which optional features are enabled."""

from fastapi import APIRouter
from pydantic import BaseModel

from app.model.db import get_engine

router = APIRouter()


class FeaturesResponse(BaseModel):
    database: bool


@router.get("/api/features", tags=["ops"], response_model=FeaturesResponse)
def features() -> FeaturesResponse:
    """List enabled features."""
    return FeaturesResponse(database=get_engine() is not None)
